plugins/pipelines/openPr.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+
+module.exports = () => ({
+    method: 'POST',
+    path: '/pipelines/openPr',
```

Comment:
```suggestion
    path: '/pipelines/{id}/openpr',
```
not sure if camel case makes sense here

Reply:
we don't really need `id` here because we don't use it

Reply:
you could take in ID instead of scmUri

Reply:
@adong what you think about this?

Reply:
I would prefer having `id` here just for logging purposes and having a consistent API URLs, 
Though I agree we don't need it for creating the PR.

Reply:
else it will de difficult when debugging to trace which URL corresponds to which pipeline creation